libs/wingsdk/src/target-tf-aws/react-app.ts

```diff
@@ -56,11 +63,9 @@ export class ReactApp extends ex.ReactApp {
       null,
       2
     )};`;
-    (this._websiteHost as Website).addFile(
-      ex.WING_JS,
-      content,
-      "text/javascript"
-    );
+    (this._websiteHost as Website).addFile(ex.WING_JS, content, {
+      contentType: "text/javascript",
+    });
```

Comment:
Can we make `_host` typed as `cloud.Website` so that we don't have to perform this type cast?